www/directives/player/mission.html

```diff
@@ -4,7 +4,7 @@
 			<p class='lead text-danger'>Mission locked</p>
 		</div>
 	</div>
-	<div ng-if='missionCtrl.missionStatus.status != "locked"'>
+	<div ng-if='missionCtrl.missionStatus.status != "locked" || true'>
```

Comment:
This to denote the temporary workaround of the PR. I can just remove the `ng-if` if you prefer